marlowe-chain-sync/deploy/block.sql

```diff
@@ -0,0 +1,19 @@
+-- Deploy chain:block to pg
+-- requires: appschema
+
+BEGIN;
+
+SET client_min_messages = 'warning';
+
+CREATE TABLE chain.block
+  ( id BYTEA NOT NULL
+  , slotNo BIGINT NOT NULL
+  , blockNo BIGINT NOT NULL
+  , rollbackToBlock BYTEA
+  , rollbackToSlot BIGINT
```

Comment:
Are we sure we don't need `rollbackToBlockNo`, too? It's probably redundant, but we should think about edge cases.

Reply:
`SlotNo` and `BlockHeaderHash` are sufficient to uniquely identify a block (and this is what we build the indexes on). Technically, `BlockHeaderHash` is sufficient, but having the slot no allows us to make use of the composite index and improve query performance further.